README.md

```diff
@@ -209,6 +212,12 @@ If you're sure Python dependences are installed correctly, please create an issu
 If you use EAF Markdown Previewer, to get consistent previewing, you need to access [Github Personal access token site](https://github.com/settings/tokens/new?scopes=), fill something in "Token description" and click button "Generate token" to get your personal token. Then set the token:
 
 ```Elisp
+;; for security reasons (if you add files to git) you can create secret.el with 
+;; following parameters and then load it (instead of setting directly inside init.el)
+;; put in the rigth path to replace, and add it to .gitignore
+;; (load "~/.emacs.d/secret.el")
+
+;; inside secret.el [or init.el if you are fine with that.]
```

Comment:
I don't it's up to EAF to teach them this.

Reply:
true that's one thing, remove it if you feel its not okay, but we should try to promote good practices right?

Reply:
Yes, another option is to call customize-variable and save their value interactively, which will be saved to `customize-set-variables.el`, and if configured correctly, shouldn't be tracked by a VCS.